src/adapters/concordance/adapter.js

```diff
@@ -63,7 +63,7 @@ class AlpheiosConcordanceAdapter extends BaseAdapter {
         return {
           wordUsageExamples: parsedWordUsageList,
           targetWord: homonym.targetWord,
-          language: homonym.language,
+          language: LanguageModelFactory.getLanguageCodeFromId(homonym.languageID),
```

Comment:
Bridget, why did you change it to the same?
```
get language () {
    console.warn(`Please use languageID instead`)
    return LMF.getLanguageCodeFromId(this.languageID)
  }
```

Reply:
because homonym.language is deprecated and throws a warning.